src/frontend/context/SøknadContext.ts

```diff
@@ -54,37 +54,55 @@ const [SøknadProvider, useSøknad] = createUseContext(() => {
         }
     }, [bruker.status]);
 
-    const erSøknadGyldig = (): boolean => {
-        let søknadenErGyldig = true;
-        if (process.env.NODE_ENV === 'development') {
-            return true;
+    const validerSøknad = (validerSøknad: ISøknadDTO): boolean => {
+        const søknadFeilmeldinger: FeiloppsummeringFeil[] = [];
+
+        if (
+            !validerSøknad.søkerMedOpplysninger.harOppholdtSegINorgeSiste12Måneder &&
+            (validerSøknad.søkerMedOpplysninger.komTilNorge === '' ||
```

Comment:
trenger man å sjekke tom string når man sjekker `!validerSøknad.søkerMedOpplysninger.komTilNorge`?

Reply:
Ja, den settes til '' når staten initialiseres.

Reply:
Lurer på om ! gir true både for undefined og "" 🤔 ikke viktig uansett, dette blir hvertfall riktig! Bra jobba 💯